# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/util-vserver/Esetgroups.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- util-vserver-0.30.216-pre3054/ensc_wrappers/wrappers-unistd.hc.orig	2013-10-31 14:44:26.000000000 +0200
+++ util-vserver-0.30.216-pre3054/ensc_wrappers/wrappers-unistd.hc	2014-03-14 19:12:34.932828751 +0200
@@ -156,7 +156,7 @@
   FatalErrnoError(setgid(gid)==-1, "setgid()");
 }
 
-#if defined(_GRP_H) && (defined(__USE_BSD) || defined(__dietlibc__))
+#if defined(_GRP_H) && (defined(__USE_BSD) || !defined(__GLIBC__))
 inline static void
 Esetgroups(size_t size, const gid_t *list)
 {

